packages/sra-spec/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
```

Comment:
Since you didn't change the contents of the package (ie. the API) , I don't think you need to add to the change log.

Reply:
Got it -- removed this